week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,9 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var sum = 15 + 8;
+var percentageOfStudents = (15 / 23) * 100;
```

Comment:
Here, we want to use variables instead of hardcoded values and do this instead: `var percentageOfStudents = (numberOfStudents / sum) * 100;`